master/internal/config/config.go

```diff
@@ -146,6 +146,7 @@ type Config struct {
 	Cache                 CacheConfig                       `json:"cache"`
 	Webhooks              WebhooksConfig                    `json:"webhooks"`
 	FeatureSwitches       []string                          `json:"feature_switches"`
+	ReservedPorts         []int                             `json:"reserved_ports"`
```

Comment:
i have a random drive-by thought.. we should and an "experimental" section so we can break the configs later when it becomes more obvious where they should live without feeling bad about it.

Reply:
So long as we aren't tempted to make everything experimental :)

Reply:
i think a lot of stuff should be at first, ha